src/main/resources/application.yaml

```diff
@@ -12,13 +12,13 @@ management:
         include: health, info, prometheus
 
 springdoc:
-  packagesToScan: uk.gov.hmcts.reform.demo.controllers
+  packagesToScan: uk.gov.hmcts.reform.rtl-export.controllers
```

Comment:
is this package exists: uk.gov.hmcts.reform.rtl-export

Reply:
Changed packagesToScan to "uk.giv.hmcts.refrom.civil.controllers".